<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat_page</class>
 <widget class="QWidget" name="chat_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>616</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { color: rgb(165, 165, 165); }
#chat_page{
	background-color: rgb(0, 0, 0);	
}
/* TOP */
#top {
	background-color: rgb(30, 32, 33);
	border-radius: 10px;
}
#user_name { 
	color: rgb(179, 179, 179);
	font: 600 12pt &quot;Segoe UI&quot;;
}
#user_image {
	border: 1px solid rgb(30, 32, 33);
	background-color: rgb(47, 48, 50);
	border-radius: 20px;
}
#top QPushButton {
	background-color: rgb(47, 48, 50);
	border-radius: 20px;
	background-repeat: no-repeat;
	background-position: center;
}
#top QPushButton:hover {
	background-color: rgb(61, 62, 65);
}
#top QPushButton:pressed {
	background-color: rgb(16, 17, 18);
}
#btn_attachment_top {	
	background-image: url(:/chat/icons_svg/images/chatlog/icons_svg/icon_attachment.svg);
}
#btn_open_folder {
	background-image: url(:/icons/images/icons/cil-folder-open.png);
}
#btn_more_top {	
	background-image: url(:/chat/icons_svg/images/chatlog/icons_svg/icon_more_options.svg);
}
/* BOTTOM */
#bottom QPushButton {
	background-color: rgb(47, 48, 50);
	border-radius: 20px;
	background-repeat: no-repeat;
	background-position: center;
}
#bottom QPushButton:hover {
	background-color: rgb(61, 62, 65);
}
#bottom QPushButton:pressed {
	background-color: rgb(16, 17, 18);
}
#send_message_frame { 
	background-color: rgb(47, 48, 50);
	border-radius: 20px;
}
#send_message_frame QPushButton {
	background-color: rgb(76, 77, 80);
	border-radius: 15px;
	background-repeat: no-repeat;
	background-position: center;
}
#send_message_frame QPushButton:hover {
	background-color: rgb(81, 82, 86);
}
#send_message_frame QPushButton:pressed {
	background-color: rgb(16, 17, 18);
}
#line_edit_message {
	background-color: transparent;
	selection-color: rgb(255, 255, 255);
	selection-background-color: rgb(149, 199, 0);
	border: none;
	padding-left: 15px;
	padding-right: 15px;
	background-repeat: none;
	background-position: left center;
	font: 10pt &quot;Segoe UI&quot;;
	color: rgb(94, 96, 100);
}
#line_edit_message:focus {
	color: rgb(165, 165, 165);
}
#btn_emoticon{
	background-image: url(:/chat/icons_svg/images/chatlog/icons_svg/icon_emoticons.svg);
}
#btn_send_message{	
	background-image: url(:/chat/icons_svg/images/chatlog/icons_svg/icon_send.svg);
}
#btn_attachment_bottom{	
	
	background-image: url(:/chat/icons_svg/images/chatlog/icons_svg/icon_more_options.svg);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="top">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="user_image">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="user_information_frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="user_name">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>User name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="user_description">
           <property name="styleSheet">
            <string notr="true">background: transparent;</string>
           </property>
           <property name="text">
            <string>User description</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="chat_log_filename">
        <property name="text">
         <string>connected last time 24h ago</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_open_folder">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_more_top">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="chat_messages">
     <property name="styleSheet">
      <string notr="true">background: transparent</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
     <widget class="QWidget" name="messages_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>959</width>
        <height>486</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent</string>
      </property>
      <layout class="QVBoxLayout" name="chat_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="messages_frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="chat_messages_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bottom">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="send_message_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_emoticon">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_edit_message">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="placeholderText">
            <string>Message #user</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_send_message">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_attachment_bottom">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
